### ISSUE TITLE
NTR: 2-hydroxyisocaproic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8582]
New term request 2-hydroxyisocaproic acid high in urine (segal_030509170849): alpha-hydroxyisocaproic acid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000601
- label: Elevated urinary 2-hydroxyisocaproic acid level
- parents: [Increased urine proteinogenic amino acid derivative level (HP:0033097)](https://hpo.jax.org/app/browse/term/HP:0033097)
- pmid: 
- def: The amount of 2-hydroxyisocaproic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8582
- synonyms:
